### PR TITLE
Export mysql schema variable (and bump go to 1.17)

### PR DESCRIPTION
**Pull request by lucasmrod**
Hi folks.

I thought it would be a good idea to export the schema as a Go variable. Our ([fleet](https://github.com/fleetdm/fleet)) migration scripts are in Go so this change would come in handy.

And the embed feature was introduced in go 1.17 (hence the change in `go.mod` to upgrade). Am guessing this is a-ok as the related project, MicroMDM, is already in 1.17 ([go.mod](https://github.com/micromdm/micromdm/blob/main/go.mod)).

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -9,3 +9,8 @@ require (
 	github.com/lib/pq v1.10.6
 	go.mozilla.org/pkcs7 v0.0.0-20210826202110-33d05740a352
 )
+
+require (
+	golang.org/x/net v0.0.0-20191009170851-d66e71096ffb // indirect
+	golang.org/x/text v0.3.0 // indirect
+)
```
Comment (jessepeterson) [inline go.mod:16]: I'd be willing to accept if these `require`'s weren't, er, required. :) Can we remove them (or does `go mod tidy` get rid of them)?

Comment (lucasmrod) [reply go.mod:16]: `go mod tidy` does not remove them. Tried removing them manually, and `go mod tidy` adds them back again.

`go mod graph` shows that both of these `// indirect` are coming from `github.com/RobotsAndPencils/buford@v0.14.0`: 
```
go mod graph
github.com/micromdm/nanomdm github.com/RobotsAndPencils/buford@v0.14.0
github.com/micromdm/nanomdm github.com/go-sql-driver/mysql@v1.6.0
github.com/micromdm/nanomdm github.com/groob/plist@v0.0.0-20220217120414-63fa881b19a5
github.com/micromdm/nanomdm github.com/lib/pq@v1.10.6
github.com/micromdm/nanomdm go.mozilla.org/pkcs7@v0.0.0-20210826202110-33d05740a352
github.com/micromdm/nanomdm golang.org/x/net@v0.0.0-20191009170851-d66e71096ffb
github.com/micromdm/nanomdm golang.org/x/text@v0.3.0
github.com/RobotsAndPencils/buford@v0.14.0 github.com/aai/gocrypto@v0.0.0-20160205191751-93df0c47f8b8
github.com/RobotsAndPencils/buford@v0.14.0 github.com/gorilla/mux@v1.7.3
github.com/RobotsAndPencils/buford@v0.14.0 golang.org/x/crypto@v0.0.0-20191011191535-87dc89f01550
github.com/RobotsAndPencils/buford@v0.14.0 golang.org/x/net@v0.0.0-20191009170851-d66e71096ffb
golang.org/x/net@v0.0.0-20191009170851-d66e71096ffb golang.org/x/crypto@v0.0.0-20190308221718-c2843e01d9a2
golang.org/x/net@v0.0.0-20191009170851-d66e71096ffb golang.org/x/sys@v0.0.0-20190215142949-d0b11bdaac8a
golang.org/x/net@v0.0.0-20191009170851-d66e71096ffb golang.org/x/text@v0.3.0
golang.org/x/crypto@v0.0.0-20191011191535-87dc89f01550 golang.org/x/net@v0.0.0-20190404232315-eb5bcb51f2a3
golang.org/x/crypto@v0.0.0-20191011191535-87dc89f01550 golang.org/x/sys@v0.0.0-20190412213103-97732733099d
golang.org/x/crypto@v0.0.0-20190308221718-c2843e01d9a2 golang.org/x/sys@v0.0.0-20190215142949-d0b11bdaac8a
golang.org/x/net@v0.0.0-20190404232315-eb5bcb51f2a3 golang.org/x/crypto@v0.0.0-20190308221718-c2843e01d9a2
golang.org/x/net@v0.0.0-20190404232315-eb5bcb51f2a3 golang.org/x/text@v0.3.0
```

>  Also I suppose we'd want to do the same with the pgsql driver.

Not sure what you mean, `_ github.com/lib/pq` is imported by `cmd/cli/cli.go`.

Comment (jessepeterson) [reply go.mod:16]: Re: import: oh! that makes sense!

Re: pgsql: I mean also adding a `Schema` go:embed variable. :)

Comment (lucasmrod) [reply go.mod:16]: > Re: pgsql: I mean also adding a Schema go:embed variable. :)

Ah!, gotcha. [c245c96](https://github.com/micromdm/nanomdm/pull/55/commits/c245c96de5dc4919adc79f0b39ebcd165ed17f23).

**Review by jessepeterson — APPROVED** [review]
Thanks!

**Pull request merged**